src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,681 @@
+            {
+                for (int iinfo = 0; iinfo < _bindings.InfoCount; iinfo++)
+                {
+                    if (!activeInfos(iinfo))
+                        continue;
+                    getters[iinfo] = MakeGetter(input, iinfo);
+                }
+                ch.Done();
+                return getters;
+            }
+        }
+
+        protected override int MapColumnIndex(out bool isSrc, int col)
+        {
+            return _bindings.MapColumnIndex(out isSrc, col);
+        }
+
+        protected override bool? ShouldUseParallelCursors(Func<int, bool> predicate)
+        {
+            return true;
+        }
+
+        protected override IRowCursor GetRowCursorCore(Func<int, bool> predicate, IRandom rand = null)
+        {
+            Host.AssertValue(predicate, "predicate");
+            Host.AssertValueOrNull(rand);
+
+            var inputPred = _bindings.GetDependencies(predicate);
+            var active = _bindings.GetActive(predicate);
+            var input = Source.GetRowCursor(inputPred, rand);
+            return new RowCursor(Host, this, input, active);
+        }
+
+        public override IRowCursor[] GetRowCursorSet(out IRowCursorConsolidator consolidator, Func<int, bool> predicate, int n, IRandom rand = null)
+        {
+            Host.CheckValue(predicate, nameof(predicate));
+            Host.CheckValueOrNull(rand);
+
+            var inputPred = _bindings.GetDependencies(predicate);
+            var active = _bindings.GetActive(predicate);
+            var inputs = Source.GetRowCursorSet(out consolidator, inputPred, n, rand);
+            Host.AssertNonEmpty(inputs);
+
+            if (inputs.Length == 1 && n > 1 && _bindings.AnyNewColumnsActive(predicate))
+                inputs = DataViewUtils.CreateSplitCursors(out consolidator, Host, inputs[0], n);
+            Host.AssertNonEmpty(inputs);
+
+            var cursors = new IRowCursor[inputs.Length];
+            for (int i = 0; i < inputs.Length; i++)
+                cursors[i] = new RowCursor(Host, this, inputs[i], active);
+            return cursors;
+        }
+
+        private interface ITensorValueGetter
+        {
+            TFTensor GetTensor();
+        }
+
+        private class TensorValueGetter<T> : ITensorValueGetter
+        {
+            private readonly ValueGetter<T> _srcgetter;
+            private readonly ColumnType _type;
+            private readonly ConstructorInfo _ctor;
+
+            public TensorValueGetter(IRow input, int colIndex)
+            {
+                _srcgetter = input.GetGetter<T>(colIndex);
+                _type = input.Schema.GetColumnType(colIndex);
+                _ctor = typeof(TFTensor).GetConstructor(new[] { typeof(T) });
+            }
+            public TFTensor GetTensor()
+            {
+                var scalar = default(T);
+                _srcgetter(ref scalar);
+                return (TFTensor)_ctor.Invoke(null, new object[] { scalar });
```

Comment:
>(TFTensor)_ctor.Invoke(null, new object[] { scalar }); [](start = 23, length = 54)

This reflection call is not intended here. We need a method in TFSharp where we can pass in generic data to TF. Currently, there are methods in TFSharp to do that but those are not exposed. I and @Eric will work on that.